server/src/main/java/io/seata/server/health/DataBaseHealthIndicator.java

```diff
@@ -0,0 +1,61 @@
+package io.seata.server.health;
```

Comment:
Error:  /home/runner/work/seata/seata/server/src/main/java/io/seata/server/health/DataBaseHealthIndicator.java:2: Line does not match expected header line of '[\*\s]Copyright 1999-2019 Seata.io Group.'. [RegexpHeader]

Reply:
fixed